src/GitVersionTask.Tests/VersionAndBranchFinderTests.cs

```diff
@@ -0,0 +1,140 @@
+﻿#region License
+
+// --------------------------------------------------
+// Copyright © OKB. All Rights Reserved.
+// 
+// This software is proprietary information of OKB.
+// USE IS SUBJECT TO LICENSE TERMS.
+// --------------------------------------------------
+
+#endregion
+
+using System;
+using System.Collections.Concurrent;
+using System.Text;
+
+using GitVersion;
+using GitVersion.Helpers;
+
+using NUnit.Framework;
+
+using Shouldly;
+
+[TestFixture]
+public class VersionAndBranchFinderTests
+{
+    [Test]
+    public void CacheFileExistsOnDisk()
```

Comment:
Unfortunately, it only affect the task, since it's done in the VersionAndBranchFinder, which for some reason aren't used by the EXE.